Network/Public/Include/Network/CombiningHandlers.hpp

```diff
@@ -14,36 +14,36 @@ namespace Network
 		public:
 		/**
 		* @brief Method for serialization, compression and encryption of outcoming messages.
-		* @param message    - buffer that contains data that should be serialized, compressed and encrypted.
-		* @param bodyBuffer - buffer that will contain serialized, compressed and encrypted body.
+		* @param source      - buffer that contains data that should be serialized, compressed and encrypted.
+		* @param destination - buffer that will contain serialized, compressed and encrypted body.
 		*/
-		MessageProcessingState operator()(const Message& message,yas::shared_buffer& bodyBuffer) const
+		static MessageProcessingState handleOutcomingMessage(const Message& source,yas::shared_buffer& destination)
```

Comment:
Name the function shortly, `make_handler`. Both functions could be named the same, but overloaded by arguments order